napalm_ios/ios.py

```diff
@@ -592,6 +592,13 @@ def get_optics(self):
             port_detail['physical_channels'] = {}
             port_detail['physical_channels']['channel'] = []
 
+            # If interface is shutdown it returns "N/A" as output power.
+            # Converting that to -40.0 float
+            try:
+                float(output_power)
+            except ValueError:
+                output_power = -40.0
```

Comment:
Yeah, I am fine with -100 as a sort of "infinite" value. Eventually we will move this to the classes in `napalm-yang` and those have support to leave empty values.

Before moving forward I'd like @mirceaulinic's opinion though but if he agrees we can probably create a few issues to (1) document the behavior in `napalm-base` and (2) update the rest of the drivers as well.